ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -592,15 +598,14 @@ export const DatasetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), wi
       // Only allow selection of genomics prepackaged concept sets if genomics
       // data extraction is possible, since extraction is the only action that
       // can be taken on genomics variant data from the dataset builder.
-      if (serverConfigStore.get().config.enableGenomicExtraction &&
-        getCdrVersion(this.props.workspace, this.props.cdrVersionTiersResponse).hasWgsData) {
+      if (serverConfigStore.get().config.enableGenomicExtraction && this.getCdrVersion().hasWgsData) {
         PREPACKAGED_DOMAINS = {
           ...PREPACKAGED_DOMAINS,
           ...PREPACKAGED_WITH_WHOLE_GENOME
         };
       }
       // Add Zipcode Socioeconomic status data if were in controlled tier dataset
-      if (getCdrVersion(this.props.workspace, this.props.cdrVersionTiersResponse).accessTierShortName === 'controlled') {
+      if (this.props.workspace.accessTierShortName === 'controlled') {
```

Comment:
not necessary to check the CDR Version for this, because the Workspace already has it.  You can trust that this will be set correctly because it's derived from the CDR Version on the backend: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java#L39